src/main/frontend/components/editor.cljs

```diff
@@ -158,29 +158,19 @@
                               (cons (first matched-pages)
                                     (cons q (rest matched-pages)))
                               (cons q matched-pages))))]
-      [:div
-       (when (and db-tag?
-                        ;; Don't display in heading
-                  (not (some->> edit-content (re-find #"^\s*#"))))
-         [:div.flex.flex-row.items-center.px-4.py-1.text-sm.opacity-70.gap-2
-          "Turn this block into a page:"
```

Comment:
Removed this since the UX is bad, most objects will be just blocks instead of pages, we can offer other ways to turn any blocks into a page, e.g. adding it to the context menu / commands, notice user to merge pages if there's another page has the same name with the current block.